frontend/src/app/side-menu/side-menu.component.ts

```diff
@@ -62,8 +72,16 @@ export class SideMenuComponent {
     this.sharedService.sendFormData(formData).subscribe((event: any) => {
       if (typeof (event) === 'object')  {
```

Comment:
two spaces after ). Can you use a formatter to clean up the spacing?

Reply:
I've run into problems installing google/gts formatter so I'm just cleaning these up manually until I can get it working